.env.custom_node

```diff
@@ -3,7 +3,7 @@ OPERATOR_ACCOUNT_ID=0.0.1022
 OPERATOR_ACCOUNT_PRIVATE_KEY=302e020100300506032b657004220420a608e2130a0a3cb34f86e757303c862bee353d9ab77ba4387ec084f881d420d4
 NODE_TYPE=local
 NODE_IP=127.0.0.1:50211
-NODE_ACCOUNT_ID=3
+NODE_ACCOUNT_ID=0.0.3
```

Comment:
consensusInfoClient.js should be changed as follows on line 7:
```js
      const node = { [process.env.NODE_IP]: AccountId.fromString(process.env.NODE_ACCOUNT_ID) };
```

Currently it does not set properly the node account

Reply:
I'm not sure how this was working for me before, but you're right. Fixed.